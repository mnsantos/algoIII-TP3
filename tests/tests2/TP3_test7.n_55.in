55 490
4 10
1 8
1 2
1 7
1 3
2 7
1 4
1 16
1 17
4 7
1 18
1 9
4 13
3 9
8 12
1 6
1 12
1 5
1 10
2 11
3 15
8 13
6 11
3 13
8 11
1 11
4 12
2 13
52 54
52 53
50 55
50 52
49 53
49 52
48 55
48 52
48 49
47 55
47 53
47 51
47 50
46 51
46 48
46 47
45 54
45 53
45 51
45 50
45 49
44 55
44 54
44 50
44 48
44 47
43 52
43 50
43 49
42 55
42 53
42 49
42 47
42 46
41 55
41 53
41 51
41 50
41 48
41 45
41 44
40 52
40 49
39 54
39 50
39 49
39 47
38 54
38 53
38 48
38 40
37 53
37 50
37 46
37 45
37 43
37 40
36 52
36 51
36 50
36 44
36 42
36 39
36 37
35 54
35 52
35 51
35 50
35 47
35 45
35 44
35 39
35 38
35 36
34 55
34 50
34 49
34 48
34 38
34 36
34 35
33 53
33 47
33 43
33 39
33 38
33 34
32 53
31 52
31 51
31 49
31 48
31 46
31 44
31 43
30 47
30 46
30 42
30 40
30 37
29 55
29 54
29 49
29 47
29 45
12 32
12 31
12 25
12 20
12 18
12 17
12 16
12 14
12 13
11 50
11 45
11 44
11 42
11 40
11 35
11 31
11 30
11 26
11 18
11 17
11 14
10 55
10 53
10 52
10 47
10 45
10 42
10 35
10 33
10 32
10 30
10 24
10 21
9 53
9 50
9 48
9 46
9 44
9 42
9 40
9 36
9 35
9 32
9 31
9 30
9 27
9 26
9 22
9 16
9 15
8 40
8 36
8 34
8 32
8 27
8 26
8 25
8 23
8 20
7 53
7 50
7 49
7 47
7 42
7 35
7 33
7 24
7 23
7 17
6 50
6 49
6 37
6 34
6 33
6 32
6 31
6 30
6 22
6 21
6 17
5 55
5 48
5 47
5 46
5 45
5 43
5 42
5 41
5 37
5 32
5 27
5 22
1 50
1 49
1 48
1 47
1 46
1 45
1 44
1 43
1 42
1 41
1 40
1 39
1 38
1 37
1 36
1 35
1 34
1 33
1 32
1 31
1 30
1 29
1 28
1 27
1 19
1 20
1 21
1 22
1 23
1 24
1 25
1 26
1 51
1 52
1 53
1 54
1 55
2 26
2 33
2 35
2 37
2 39
2 45
2 47
2 48
2 51
2 52
2 54
3 31
3 34
3 35
3 36
3 39
3 43
3 44
3 45
3 48
3 53
3 54
4 34
4 36
4 38
4 43
4 46
4 53
4 54
5 17
12 33
12 34
12 37
12 38
12 44
12 46
12 47
12 48
12 50
12 53
12 55
13 19
13 23
13 24
13 27
13 28
13 35
13 37
13 40
13 44
13 46
13 49
13 55
14 18
14 19
14 31
14 39
14 40
14 42
14 43
14 45
14 49
14 51
15 17
15 20
15 26
15 29
15 32
15 36
15 41
15 43
15 49
15 53
15 55
16 20
16 26
16 28
16 29
16 30
16 33
16 35
16 37
16 38
16 44
16 45
16 48
16 50
16 54
17 22
17 30
17 34
17 38
17 44
17 48
17 49
17 52
17 54
18 19
18 20
18 23
18 32
18 35
18 40
18 42
18 46
18 49
18 50
18 51
19 21
19 22
19 23
19 30
19 33
19 35
19 39
19 41
19 47
19 49
19 50
19 54
19 55
20 33
20 34
20 36
20 39
20 42
20 43
20 44
20 46
20 47
20 49
20 54
20 55
21 22
21 24
21 27
21 28
21 34
21 36
21 39
21 41
21 44
21 50
21 51
21 52
21 55
22 24
22 27
22 32
22 34
22 35
22 38
22 39
22 44
22 46
22 49
22 54
23 24
23 26
23 27
23 43
23 44
23 48
23 50
23 55
24 25
24 27
24 29
24 35
24 36
24 41
24 42
24 43
24 44
24 45
24 47
24 48
24 50
25 27
25 29
25 35
25 39
25 41
25 43
25 44
25 45
25 47
25 51
25 52
26 31
26 35
26 36
26 38
26 39
26 49
26 50
26 54
26 55
27 41
27 42
27 45
27 47
27 49
27 50
27 51
27 52
28 35
28 36
28 37
28 39
28 41
28 44
28 46
28 50
28 51
28 53
29 30
29 32
29 38
29 39
29 43
29 44
1 15
1 13
1 14
3 5
2 5
3 16
0
