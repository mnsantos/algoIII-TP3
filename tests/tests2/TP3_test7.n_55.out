146 6 1 24 35 47 44 50
